### PR TITLE
Call of Yucutan 翻译提交

### PR DESCRIPTION
**Pull request by Huantanhua**

### CODE REVIEW

**Comment by Huantanhua** [comment]
> 您好,非常荣幸能够跟您在同一时间进行同一模组的翻译。相信通过交流,我们可以得出一个更加精细,严谨而又具有生机的译本。诸多我个人的翻译思路和意见会以评论的形式在您的文本中加以提出,请您过目。

辛苦了您了

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json
/sort-keys projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/en_us.json

**Comment by luoxueyasha** [comment]
两份pr只能说是差异很大……你们在code review里先把差异讨论出来解决了吧
<details>
<summary>PR间差异对比</summary>
|行号|Key|en 文件|#4430 的 cn 文件|#4431 的 cn 文件|
|:----|:----|:----|:----|:----|
|2|advancements.defeat_boss.descr|Defeat an Ancient God|打败一位远古神明|克神制胜|
|3|advancements.defeat_boss.title|God Slayer|屠神者|弑神者|
|4|advancements.get_burnt_by_light.descr|Stand in front of a Solar Ray|站在日光射线前面|接受日炎光束直射|
|5|advancements.get_burnt_by_light.title|Danger: High Voltage|高压危险|当心高压电|
|6|advancements.interact_with_mirror.descr|Interact with a Reflector|与反射器交互|与反光器交互|
|8|advancements.kill_villager_with_macu.descr|Kill a Villager using a Macuahuitl or Obsidian Tecpatl|利用黑曜石锯剑或黑曜石匕首杀死一个村民|使用黑曜石剑或黑曜石匕首杀死一个村民|
|9|advancements.kill_villager_with_macu.title|Blood for the Blood God|活祭血神|血祭血神|
|10|advancements.obtain_blessing.descr|Obtain a Blessing by making 10 sacrifices to an Idol|向神像献祭10次以获得祝福|对雕像献祭十次并得到赐福|
|11|advancements.obtain_blessing.title|Divine Favor|天赐恩宠|神明的眷顾|
|12|advancements.obtain_jade_armor.descr|Obtain a full set of Jade armor|获得翡翠套装|获得一整套玉盔甲|
|13|advancements.obtain_jade_armor.title|Looks Like Emerald to Me|看起来很像绿宝石|看起来就像绿宝石一样|
|14|advancements.sacrifice_items.descr|Sacrifice an item to an Idol|给神像献祭一个物品|向雕像献祭一个物品|
|15|advancements.sacrifice_items.title|I Want To Believe|我要相信|吾欲信从|
|16|advancements.shoot_back_blow_gun.descr|Use a Blow Gun against a Blowgun Huracan|利用吹筒来对抗胡拉坎|使用吹箭筒射击吹箭胡拉坎|
|17|advancements.shoot_back_blow_gun.title|no u|以牙还牙|其人之身|
|18|advancements.use_hematite_shield.descr|Redirect Solar Rays remotely with your Hematite Shield|利用赤铁盾远程改变日光射线的方向|使用赤铁盾牌偏折日炎光束|
|19|advancements.use_hematite_shield.title|I Have Become Light|我会指引你|妈妈，我变成光了|
|20|advancements.use_mayan_calendar.descr|Obtain a Mayan Calendar to keep up with Celestial Events|获得一个玛雅日历以跟踪月相变化|获得一个玛雅日历并关注天体异变|
|21|advancements.use_mayan_calendar.title|Looking for the Moon|追寻月亮|追星赶月|
|22|block.call_of_yucutan.ah_puch_idol|§dAh Puch Idol|§d阿·普切神像|§d阿·普切雕像|
|23|block.call_of_yucutan.ancient_gold|§eAncient Gold|§e古代金块|§e陈金锭|
|24|block.call_of_yucutan.ancient_gold_pillar|Ancient Gold Pillar|古代金柱|陈金柱|
|25|block.call_of_yucutan.ancient_gold_slab|Ancient Gold Slab|古代金台阶|陈金台阶|
|26|block.call_of_yucutan.ancient_gold_stairs|Ancient Gold Stairs|古代金楼梯|陈金楼梯|
|27|block.call_of_yucutan.ancient_gold_wall|Ancient Gold Wall|古代金墙|陈金墙|
|28|block.call_of_yucutan.bone_torch_ground|Bone Torch|骨火把|陈旧之骨火把|
|29|block.call_of_yucutan.bone_torch_wall|Bone Torch|骨火把|陈旧之骨火把|
|30|block.call_of_yucutan.chiseled_jade|Chiseled Jade|錾制翡翠块|雕纹玉块|
|31|block.call_of_yucutan.chiseled_red_limestone|Chiseled Red Limestone|錾制红色石灰岩|雕纹红石灰石|
|32|block.call_of_yucutan.deepslate_jade_ore|Deepslate Jade Ore|深层翡翠矿石|深层玉矿石|
|33|block.call_of_yucutan.golden_torch_ground|Golden Torch|金火把|陈金火把|
|34|block.call_of_yucutan.golden_torch_wall|Golden Torch|金火把|陈金火把|
|35|block.call_of_yucutan.hematite_block|Hematite Block|赤铁块|赤铁矿石|
|36|block.call_of_yucutan.jade_block|Jade Block|翡翠块|玉块|
|37|block.call_of_yucutan.jade_ore|Jade Ore|翡翠矿石|玉矿石|
|38|block.call_of_yucutan.jade_slab|Jade Slab|翡翠：台阶|玉台阶|
|39|block.call_of_yucutan.jade_stairs|Jade Stairs|翡翠楼梯|玉台阶|
|40|block.call_of_yucutan.jade_tile_slab|Jade Tile Slab|翡翠瓦台阶|玉砖台阶|
|41|block.call_of_yucutan.jade_tile_stairs|Jade Tile Stairs|翡翠瓦楼梯|玉瓦楼梯|
|42|block.call_of_yucutan.jade_tiles|Jade Tiles|翡翠瓦|玉瓦|
|43|block.call_of_yucutan.jade_wall|Jade Wall|翡翠墙|玉墙|
|44|block.call_of_yucutan.kukulkan_idol|§dKukulkan Idol|§d库库尔坎神像|§d库库尔坎雕像|
|45|block.call_of_yucutan.mossy_limestone_tiles|Mossy Red Limestone Tiles|生苔红色石灰岩瓦|红色石灰苔石瓦|
|46|block.call_of_yucutan.mossy_red_limestone|Mossy Red Limestone|生苔红色石灰岩|红色石灰苔石|
|47|block.call_of_yucutan.mossy_ritual_rock_tiles|Mossy Ritual Rock Tiles|生苔仪式石瓦|仪式苔岩瓦|
|48|block.call_of_yucutan.mossy_ritual_rocks|Mossy Ritual Rocks|仪式苔石|仪式苔岩|
|49|block.call_of_yucutan.mystic_vine_block|Mystic Vines|神秘藤蔓块|神秘藤|
|50|block.call_of_yucutan.overgrown_chest|Overgrown Chest|覆草箱子|藤蔓寄生的箱子|
|51|block.call_of_yucutan.polished_jade_block|Polished Jade Block|磨制翡翠块|磨制玉块|
|52|block.call_of_yucutan.polished_red_limestone|Polished Red Limestone|磨制红色石灰岩|磨制红色石灰石|
|53|block.call_of_yucutan.polished_red_limestone_slab|Polished Red Limestone Slab|磨制红色石灰岩台阶|磨制红色石灰石台阶|
|54|block.call_of_yucutan.polished_red_limestone_stairs|Polished Red Limestone Stairs|磨制红色石灰岩楼梯|磨制红色石灰石楼梯|
|55|block.call_of_yucutan.polished_red_limestone_wall|Polished Red Limestone Wall|磨制红色石灰岩墙|磨制红色石灰石墙|
|56|block.call_of_yucutan.polished_ritual_rock|Polished Ritual Rock|磨制仪式石|磨制仪式岩|
|57|block.call_of_yucutan.polished_ritual_rock_slab|Polished Ritual Rock Slab|磨制仪式石台阶|磨制仪式岩台阶|
|58|block.call_of_yucutan.polished_ritual_rock_stairs|Polished Ritual Rock Stairs|磨制仪式石楼梯|磨制仪式岩楼梯|
|59|block.call_of_yucutan.polished_ritual_rock_wall|Polished Ritual Rock Wall|磨制仪式石墙|磨制仪式岩墙|
|60|block.call_of_yucutan.red_limestone|Red Limestone|红色石灰岩|红色石灰石|
|61|block.call_of_yucutan.red_limestone_pressure_plate|Red Limestone Pressure Plate|红色石灰岩压力板|红色石灰石压力板|
|62|block.call_of_yucutan.red_limestone_slab|Red Limestone Slab|红色石灰岩台阶|红色石灰石台阶|
|63|block.call_of_yucutan.red_limestone_stairs|Red Limestone Stairs|红色石灰岩楼梯|红色石灰石楼梯|
|64|block.call_of_yucutan.red_limestone_tile_slab|Red Limestone Tile Slab|红色石灰岩瓦台阶|红色石灰石瓦台阶|
|65|block.call_of_yucutan.red_limestone_tile_stairs|Red Limestone Tile Stairs|红色石灰岩瓦楼梯|红色石灰石砖楼梯|
|66|block.call_of_yucutan.red_limestone_tiles|Red Limestone Tiles|红色石灰岩瓦|红色石灰石瓦|
|67|block.call_of_yucutan.red_limestone_wall|Red Limestone Wall|红色石灰岩墙|红色石灰石墙|
|68|block.call_of_yucutan.reflector|Reflector|反射器|反光器|
|69|block.call_of_yucutan.ritual_rock_pillar|Ritual Rock Pillar|仪式石柱|仪式岩柱|
|70|block.call_of_yucutan.ritual_rock_slab|Ritual Rock Slab|仪式石台阶|仪式岩台阶|
|71|block.call_of_yucutan.ritual_rock_stairs|Ritual Rock Stairs|仪式石楼梯|仪式岩楼梯|
|72|block.call_of_yucutan.ritual_rock_tiles|Ritual Rock Tiles|仪式石瓦|仪式岩瓦|
|73|block.call_of_yucutan.ritual_rock_wall|Ritual Rock Wall|仪式石墙|仪式岩墙|
|74|block.call_of_yucutan.ritual_rocks|Ritual Rocks|仪式石|仪式岩|
|75|block.call_of_yucutan.solar_projector|Solar Projector|日光投影仪|日炎投射器|
|76|death.attack.obsidian_arrow_damage|%1$s was pierced by an Obsidian Arrow|%1$s被黑曜石箭射穿|%1$s 被黑曜石箭矢刺穿了|
|77|death.attack.obsidian_arrow_damage.item|%1$s was pierced by %2$s using %3$s|%1$s被%2$s用%3$s刺穿|%1$s 被 %2$s 使用 %3$s 刺穿了|
|78|death.attack.obsidian_arrow_damage.player|%1$s was pierced whilst trying to escape %2$s|%1$s在试图逃离%2$s时被刺穿|%1$s 在试图逃离 %2$s 时被刺穿了|
|81|enchantment.call_of_yucutan.double_edged|Double Edged|双刃剑|双刃|
|82|enchantment.call_of_yucutan.double_edged.desc|Randomly damage the target twice on hit|击中目标时随机造成两次伤害|攻击有概率造成双倍伤害|
|83|enchantment.call_of_yucutan.ritual_slaught|Ritual Slaught|人祭|献祭仪式|
|84|enchantment.call_of_yucutan.ritual_slaught.desc|Deals more damages to Villagers, Illagers and Players|对村民、灾厄村民和玩家造成更多伤害|对玩家，灾厄村民和村民造成更多伤害|
|86|enchantment.call_of_yucutan.toxic_blade.desc|Inflict Poison on hit|对被攻击者施加中毒效果|攻击造成中毒效果|
|88|entity.call_of_yucutan.ah_puch.title_0|Ah Puch, the Lord of Death|阿·普切，冥府之主|阿·普切，死亡之神|
|89|entity.call_of_yucutan.ah_puch.title_1|Ah Puch, the Smelly Lord|阿·普切，腐尸之主|阿·普切，恶臭之神|
|90|entity.call_of_yucutan.ah_puch.title_2|Ah Puch, the Emaciated Lord|阿·普切，消瘦之主|阿·普切，饥萎之神|
|92|entity.call_of_yucutan.golden_guard|Golden Guard|金傀儡|陈金守卫|
|95|entity.call_of_yucutan.mitnal_monkey|Mitnal Monkey|米特纳尔猴子|冥猴|
|96|entity.call_of_yucutan.undead_warrior|Mayan Warrior|玛雅战士|玛雅武士|
|100|item.call_of_yucutan.ancient_gold_ingot|Ancient Gold Ingot|古代金锭|陈金锭|
|101|item.call_of_yucutan.blow_gun|Blow Gun|吹筒|吹箭筒|
|103|item.call_of_yucutan.bone_torch|Bone Torch|骨火把|陈旧之骨火把|
|104|item.call_of_yucutan.elder_bone|Elder Bone|陈旧骨头|陈旧的骨头|
|105|item.call_of_yucutan.fist_of_doom|Fist Of Doom|灭世拳套|灾厄之拳|
|106|item.call_of_yucutan.flint_spear|Flint Spear|燧石长矛|燧石矛|
|107|item.call_of_yucutan.fragment_of_death|§dFragment Of Death|§d死亡碎片|§d死亡之残片|
|108|item.call_of_yucutan.fragment_of_rebirth|§dFragment Of Rebirth|§d重生碎片|§d重生之残片|
|109|item.call_of_yucutan.golden_guard_spawn_egg|Golden Guard Spawn Egg|金傀儡刷怪蛋|陈金守卫刷怪蛋|
|110|item.call_of_yucutan.golden_torch|Golden Torch|金火把|陈金火把|
|111|item.call_of_yucutan.hanging_vine_hook|Hanging Vine Hook|藤蔓挂钩|藤挂钩|
|113|item.call_of_yucutan.hematite_shield|§eHematite Shield|§e赤铁盾|§e赤铁盾牌|
|115|item.call_of_yucutan.jade|Jade|翡翠|玉|
|116|item.call_of_yucutan.jade_amulet|Jade Amulet|翡翠护身符|玉护符|
|117|item.call_of_yucutan.jade_axe|Jade Axe|翡翠斧|玉斧|
|118|item.call_of_yucutan.jade_hoe|Jade Hoe|翡翠锄|玉锄|
|119|item.call_of_yucutan.jade_pickaxe|Jade Pickaxe|翡翠镐|玉稿|
|120|item.call_of_yucutan.jade_shovel|Jade Shovel|翡翠锹|玉铲|
|121|item.call_of_yucutan.jade_sword|Jade Sword|翡翠剑|玉剑|
|122|item.call_of_yucutan.jades_boots|Jade Boots|翡翠靴子|玉靴子|
|123|item.call_of_yucutan.jades_chestplate|Jade Chestplate|翡翠胸甲|玉胸甲|
|124|item.call_of_yucutan.jades_helmet|Jade Helmet|翡翠头盔|玉头盔|
|125|item.call_of_yucutan.jades_leggings|Jade Leggings|翡翠护腿|玉护腿|
|127|item.call_of_yucutan.macuahuitl|Macuahuitl|黑曜石锯剑|黑曜石剑|
|129|item.call_of_yucutan.mitnal_monkey_spawn_egg|Mitnal Monkey Spawn Egg|米特纳尔猴子刷怪蛋|冥猴刷怪蛋|
|130|item.call_of_yucutan.monkey_helmet|§eMonkey Mask|§e猴子面具|§e冥猴面具|
|135|item.call_of_yucutan.mystic_vine|Mystic Vine|神秘藤蔓|神秘藤|
|137|item.call_of_yucutan.obsidian_spear|Obsidian Spear|黑曜石长矛|黑曜石矛|
|139|item.call_of_yucutan.poisonous_dart|Poisonous Dart|毒镖|有毒吹箭|
|142|item.call_of_yucutan.undead_warrior_spawn_egg|Mayan Warrior Spawn Egg|玛雅战士刷怪蛋|玛雅武士刷怪蛋|
|143|item.call_of_yucutan.warrior_helmet|§eWarrior Mask|§e战士面具|§e玛雅武士面具|
|144|item.call_of_yucutan.wooden_spear|Wooden Spear|木长矛|木矛|
|145|item.call_of_yucutan.wooden_tecpatl|Wooden Tecpatl|木长矛|木匕首|
|146|subtitles.ah_puch_ambient|Ah Puch ambient|阿·普切：密语|阿·普切：嘶吼|
|149|subtitles.ah_puch_laugh|Ah Puch laugh|阿·普切：嘲笑|阿·普切：大笑|
|151|subtitles.ah_punch_charge|Ah Punch Charge|灭世拳套：充满|阿·普切：蓄力|
|152|subtitles.dart_shoot|Dart shoot|毒镖：发射|吹箭：发射|
|153|subtitles.echoes_of_xibalba|Echoes of Xibalba|西瓦尔巴的回响|冥界回响|
|154|subtitles.god_offering|God Offering|神像：触怒|神明：赐福|
|155|subtitles.god_sacrifice|God sacrifice|神像：献祭|神明：献祭|
|156|subtitles.kukulkan_ambient|Kukulkan ambient|库库尔坎：密语|库库尔坎：嘶吼|
|160|subtitles.light_beam|Light beam|光束：照射|光束：嗡嗡|
|161|subtitles.mayan_warrior_death|Mayan death|玛雅战士：死亡|玛雅武士：死亡|
|162|subtitles.mayan_warrior_hurt|Mayan hurt|玛雅战士：受伤|玛雅武士：受伤|
|163|subtitles.mayan_warrior_step|Mayan step|玛雅战士：迈步|玛雅武士：脚步声|
|164|subtitles.monkey_summoning|Monkey summoning|阿·普切：召唤猴子|冥猴：被召唤|
|79|effect.call_of_yucutan.ah_puch_bless|Ah Puch's Bless|阿·普切's的祝福|Ah Puch's Bless|
|80|effect.call_of_yucutan.kukulkan_bless|Kukulkan's Bless|库库尔坎's的祝福|Kukulkan's Bless|
|97|entity.call_of_yucutan.updater|Updater|更新器|Updater|
|品|Sacrifice an item to an Idol|向雕像献祭一个物品|
|140|entity.call_of_yucutan.golden_guard|Golden Guard|金傀儡|Golden Guard|陈金守卫|
|141|item.call_of_yucutan.jade_axe|Jade Axe|翡翠斧|Jade Axe|玉斧|
|142|subtitles.echoes_of_xibalba|Echoes of Xibalba|西瓦尔巴的回响|Echoes of Xibalba|冥界回响|
|143|advancements.defeat_boss.title|God Slayer|屠神者|God Slayer|弑神者|
|4|block.call_of_yucutan.polished_red_limestone|Polished Red Limestone|磨制红色石灰岩|Polished Red Limestone|磨制红色石灰石|
|146|block.call_of_yucutan.jade_tiles|Jade Tiles|翡翠瓦|Jade Tiles|玉瓦|
|10|block.call_of_yucutan.red_limestone|Red Limestone|红色石灰岩|Red Limestone|红色石灰石|
|149|item.call_of_yucutan.jade_shovel|Jade Shovel|翡翠锹|Jade Shovel|玉铲|
|151|block.call_of_yucutan.jade_ore|Jade Ore|翡翠矿石|Jade Ore|玉矿石|
|154|item.call_of_yucutan.jades_chestplate|Jade Chestplate|翡翠胸甲|Jade Chestplate|玉胸甲|
|155|item.call_of_yucutan.fragment_of_rebirth|§dFragment Of Rebirth|§d重生碎片|§dFragment Of Rebirth|§d重生之残片|
|156|block.call_of_yucutan.bone_torch_ground|Bone Torch|骨火把|Bone Torch|陈旧之骨火把|
|157|item.call_of_yucutan.jades_helmet|Jade Helmet|翡翠头盔|Jade Helmet|玉头盔|
|159|block.call_of_yucutan.jade_block|Jade Block|翡翠块|Jade Block|玉块|
|160|item.call_of_yucutan.jade_hoe|Jade Hoe|翡翠锄|Jade Hoe|玉锄|
|161|advancements.obtain_jade_armor.descr|Obtain a full set of Jade armor|获得翡翠套装|Obtain a full set of Jade armor|获得一整套玉盔甲|
|162|block.call_of_yucutan.mossy_limestone_tiles|Mossy Red Limestone Tiles|生苔红色石灰岩瓦|Mossy Red Limestone Tiles|红色石灰苔石瓦|
|163|block.call_of_yucutan.ritual_rocks|Ritual Rocks|仪式石|Ritual Rocks|仪式岩|
|164|advancements.use_hematite_shield.title|I Have Become Light|我会指引你|I Have Become Light|妈妈，我变成光了|
|165|subtitles.light_beam|Light beam|光束：照射|Light beam|光束：嗡嗡|
</details>

**Comment by ljmayor111** [comment]
除去上文提出的两项以外，对于剩下的条目我倾向于@Huantanhua的译本。

**Comment by luoxueyasha** [comment]
> 除去上文提出的两项以外

上文中提出的两项指？

**Comment by ljmayor111** [comment]
> > 除去上文提出的两项以外
> 
> 上文中提出的两项指？

您好，相关项已经在楼主的commits中进行注释，主要关于mitnal monkey-冥猴的翻译与\u0027s一项的删除。

**Comment by qznfbnj** [comment]
> > > 除去上文提出的两项以外
> > 
> > 
> > 上文中提出的两项指？
> 
> 您好，相关项已经在楼主的commits中进行注释，主要关于mitnal monkey-冥猴的翻译与\u0027s一项的删除。

没看见啊，在哪里呢？不会是还处于 Pending 状态吧？

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  "subtitles.dart_shoot": "毒镖：发射",
+  "item.call_of_yucutan.mayan_calendar": "§e玛雅日历",
+  "item.call_of_yucutan.mitnal_monkey_spawn_egg": "米特纳尔猴子刷怪蛋",
+  "block.call_of_yucutan.red_limestone_wall": "红色石灰岩墙",
+  "item.call_of_yucutan.blowgun_huracan_spawn_egg": "胡拉坎刷怪蛋",
+  "block.call_of_yucutan.golden_torch_wall": "金火把",
+  "item.call_of_yucutan.obsidian_spear": "黑曜石长矛",
+  "advancements.shoot_back_blow_gun.title": "以牙还牙",
+  "subtitles.ah_puch_death": "阿·普切：死亡",
+  "effect.call_of_yucutan.kukulkan_bless": "库库尔坎\u0027s的祝福",
```
[the remark this answers is not in the document]
Comment (luoxueyasha) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json]: 这位pr提交者的老习惯了，批替不管`'s`的

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  "block.call_of_yucutan.polished_red_limestone_slab": "磨制红色石灰岩台阶",
+  "subtitles.kukulkan_hurt": "库库尔坎：受伤",
+  "block.call_of_yucutan.ah_puch_idol": "§d阿·普切神像",
+  "entity.call_of_yucutan.updater": "更新器",
+  "block.call_of_yucutan.ancient_gold_slab": "古代金台阶",
+  "entity.call_of_yucutan.undead_warrior": "玛雅战士",
+  "block.call_of_yucutan.red_limestone_pressure_plate": "红色石灰岩压力板",
+  "item.call_of_yucutan.jade_pickaxe": "翡翠镐",
+  "subtitles.god_sacrifice": "神像：献祭",
+  "subtitles.monkey_summoning": "阿·普切：召唤猴子",
```
[the remark this answers is not in the document]
Comment (luoxueyasha) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json]: 冥猴+1
其实音译/意译都可以，特别是mitnal在中文语境里更多被音译的情况下
个人从简洁角度更喜欢意译

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
+{
+  "advancements.defeat_boss.descr": "打败一位远古神明",
+  "advancements.defeat_boss.title": "弑神者",
+  "advancements.get_burnt_by_light.descr": "站在日光射线前面",
+  "advancements.get_burnt_by_light.title": "高压危险",
+  "advancements.interact_with_mirror.descr": "与反射器交互",
+  "advancements.interact_with_mirror.title": "无线网络",
+  "advancements.kill_villager_with_macu.descr": "利用黑曜石锯剑或黑曜石匕首杀死一个村民",
+  "advancements.kill_villager_with_macu.title": "活祭血神",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:9]:
```suggestion
  "advancements.kill_villager_with_macu.title": "血祭血神",
```
原文`Blood for the Blood God`

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  "advancements.obtain_blessing.descr": "向神像献祭10次以获得祝福",
+  "advancements.obtain_blessing.title": "天赐恩宠",
+  "advancements.obtain_jade_armor.descr": "获得翡翠套装",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:12]: 是一整套翡翠套，还是一件就行
建议参考原版的

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+  "advancements.obtain_jade_armor.title": "看起来很像绿宝石",
+  "advancements.sacrifice_items.descr": "给神像献祭一个物品",
+  "advancements.sacrifice_items.title": "我要相信",
+  "advancements.shoot_back_blow_gun.descr": "利用吹筒来对抗胡拉坎",
+  "advancements.shoot_back_blow_gun.title": "以牙还牙",
+  "advancements.use_hematite_shield.descr": "利用赤铁盾远程改变日光射线的方向",
+  "advancements.use_hematite_shield.title": "我会指引你",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:19]: 原文`I Have Become Light`
没有在玩奥特曼的梗吧……？

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  "advancements.use_mayan_calendar.title": "追寻月亮",
+  "block.call_of_yucutan.ah_puch_idol": "§d阿·普切神像",
+  "block.call_of_yucutan.ancient_gold": "§e古代金块",
+  "block.call_of_yucutan.ancient_gold_pillar": "古代金柱",
+  "block.call_of_yucutan.ancient_gold_slab": "古代金台阶",
+  "block.call_of_yucutan.ancient_gold_stairs": "古代金楼梯",
+  "block.call_of_yucutan.ancient_gold_wall": "古代金墙",
+  "block.call_of_yucutan.bone_torch_ground": "骨火把",
+  "block.call_of_yucutan.bone_torch_wall": "骨火把",
+  "block.call_of_yucutan.chiseled_jade": "錾制翡翠块",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:30]: 1.20了 雕纹了

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+  "block.call_of_yucutan.ritual_rock_tiles": "仪式石瓦",
+  "block.call_of_yucutan.ritual_rock_wall": "仪式石墙",
+  "block.call_of_yucutan.ritual_rocks": "仪式石",
+  "block.call_of_yucutan.solar_projector": "日光投影仪",
+  "death.attack.obsidian_arrow_damage": "%1$s被黑曜石箭射穿",
+  "death.attack.obsidian_arrow_damage.item": "%1$s被%2$s用%3$s刺穿",
+  "death.attack.obsidian_arrow_damage.player": "%1$s在试图逃离%2$s时被刺穿",
+  "effect.call_of_yucutan.ah_puch_bless": "阿·普切的祝福",
+  "effect.call_of_yucutan.kukulkan_bless": "库库尔坎的祝福",
+  "enchantment.call_of_yucutan.double_edged": "双刃剑",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:81]: 什么效果？
斧头等不是剑的也能附魔吗

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+  "item.call_of_yucutan.undead_warrior_spawn_egg": "玛雅战士刷怪蛋",
+  "item.call_of_yucutan.warrior_helmet": "§e战士面具",
+  "item.call_of_yucutan.wooden_spear": "木长矛",
+  "item.call_of_yucutan.wooden_tecpatl": "木长矛",
+  "subtitles.ah_puch_ambient": "阿·普切：密语",
+  "subtitles.ah_puch_death": "阿·普切：死亡",
+  "subtitles.ah_puch_hurt": "阿·普切：受伤",
+  "subtitles.ah_puch_laugh": "阿·普切：嘲笑",
+  "subtitles.ah_puch_shockwave": "阿·普切：冲击波",
+  "subtitles.ah_punch_charge": "灭世拳套：充满",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:154]: huh
原文：`Ah Punch Charge`

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+  "subtitles.kukulkan_ambient": "库库尔坎：密语",
+  "subtitles.kukulkan_attack": "库库尔坎：攻击",
+  "subtitles.kukulkan_death": "库库尔坎：死亡",
+  "subtitles.kukulkan_hurt": "库库尔坎：受伤",
+  "subtitles.light_beam": "光束：照射",
+  "subtitles.mayan_warrior_death": "玛雅战士：死亡",
+  "subtitles.mayan_warrior_hurt": "玛雅战士：受伤",
+  "subtitles.mayan_warrior_step": "玛雅战士：迈步",
+  "subtitles.mitnal_punch": "灭世拳套：毁灭一击",
+  "subtitles.monkey_summoning": "阿·普切：召唤冥猴",
```
Comment (luoxueyasha) [inline projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:174]: 原文`Monkey summoning`
没提到阿普切，是只有阿普切战斗时才会播放吗

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 7 above)
Comment (Huantanhua) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:81]: 效果就是下一行写的，匕首的专属附魔

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 8 above)
Comment (Huantanhua) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:154]: 这个音效就是全套充满能以后发出的

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 5 above)
Comment (Huantanhua) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:19]: 应该没有

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 4 above)
Comment (Huantanhua) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:12]: 是套装

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 9 above)
Comment (Huantanhua) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:174]: 是的，半血一下会召唤猴儿们

File: projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json (hunk 4 above)
Comment (luoxueyasha) [reply projects/1.20/assets/call-of-yucutan/call_of_yucutan/lang/zh_cn.json:12]:
```suggestion
  "advancements.obtain_jade_armor.descr": "获得一整套翡翠盔甲",
```
致敬传奇成就`残骸裹身`

**Review by luoxueyasha — APPROVED** [review]

**Pull request merged**